osu.Game/Localisation/EditorStrings.cs

```diff
@@ -35,9 +35,14 @@ public static class EditorStrings
         public static LocalisableString SetPreviewPointToCurrent => new TranslatableString(getKey(@"set_preview_point_to_current"), @"Set preview point to current time");
 
         /// <summary>
-        /// "Export package"
+        /// "For editing (.olz)"
         /// </summary>
-        public static LocalisableString ExportPackage => new TranslatableString(getKey(@"export_package"), @"Export package");
+        public static LocalisableString ExportForEditing => new TranslatableString(getKey(@"export_for_editing"), @"For editing (.olz)");
```

Comment:
I've changed the terminology here. Reasoning is that I don't want to use "new"/"legacy" terminology, and this is the best way I can find to phrase things in a sensible way. Only other I can think of is "Lossless" for "For preservation", but I think "For editing" is most user friendly. It lets a user know that they should always be using `olz` if possible, because it is the preferred method for sharing for editing purposes.

I can understand if there may be a view that "for editing is confusing because what if i'm editing in stable?", but stable mappers know what format stable supports. Worst case they will try olz and it will fail and they'll understand.